src/smartTexture.tbasicu

```diff
@@ -0,0 +1,90 @@
+Uses "TBGL", "TImage", "File"
```

Comment:
Thanks for using this unit.

I have a little tip:
You don't have to put this file to repo, you can use directly this in your script (=magic):

```
#include "https://raw.githubusercontent.com/petrSchreiber/SmartTexture/master/smartTexture.tbasicu"
```

ThinBASIC will include it from GitHub directly :]

Reply:
I knew, but not always online, so...